RT-8: Оформить репозиторий

timoninas opened the pull request.

timoninas committed:
Add img dir

timoninas committed:
Add quote img

timoninas committed:
RT-8: Добавить контент для Readme

timoninas committed:
RT-8: Исправить роли в команде

timoninas committed:
RT-8: Исправить описание, добавить демо, запуск

Betchika99 commented on README.md:
```diff
@@ -1,2 +1,52 @@
+</p>
+
+
+# Redioteka
```
давай вот это уберем, потому что название есть на картинке, а потом в описании

Betchika99 commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+## Описание
+
+Redioteka — это стриминговый сервис, позволяющий смотреть интересующие фильмы и сериалы зарегестрированным пользователям. У всех пользователей есть возможность оформить подписку на интересующий их контент. А также оставлять оценки, подбирать фильмы и сериалы по жанрам и актерам.
```
зарегистрированным - опечатка

Betchika99 replied:
что означает "подбирать"? это рекомендательная система будет на нейроночках?)) за каждое написанное слово придется делать фичу, лучше писать аккуратнее

timoninas replied:
В том смысле, что пользователь может зайти в любой жанр и среди представленных фильмов выбрать какой он захочет. Наверное, в этом смысле употреблено слово "подбирать"

Betchika99 commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+Redioteka — это стриминговый сервис, позволяющий смотреть интересующие фильмы и сериалы зарегестрированным пользователям. У всех пользователей есть возможность оформить подписку на интересующий их контент. А также оставлять оценки, подбирать фильмы и сериалы по жанрам и актерам.
+
+Вы можете смотреть сколько угодно фильмов и сериалов без рекламы по фиксированной цене!
```
смотри, сейчас в описании идет смешение лиц, к которым оно обращается. Сначала ты пишешь как бы в общем "у всех пользователей", а зачем "Вы можете", т.е. обращаешься прямо ко мне, как к пользаку. Давай определимся со стилем. Мне больше нравится "продающее" описание вида "Здесь вы можете с легкостью оформить подписку на контент, который подходит именно вам, чтобы не пропустить новые видео!" и тому подобное

Betchika99 replied:
также в описании нужно особенно выделить вашу фичу и чем вы отличаетесь от других сервисов

Betchika99 commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+**Плюшки:** 
+
+<ol> 
```
давай список уберем и напишем "В разработке" или "To be continued.."

Betchika99 commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+## Как запустить проект
+
+(Добавить позже)
```
тоже самое "В разработке" или "To be continued.."

Betchika99 commented on README.md:
```diff
@@ -1,2 +1,52 @@
+
+## Демо
+
+[redioteka.com](https://www.netflix.com/ru/)
```
ссылка забавная конечно))0)

timoninas replied:
Вставил ссылку, чтобы хоть как-то заполнить пустоту 🙃

Betchika99 reviewed (changes requested):
В целом очень хорошо. Нужно оставить еще место для API и добавить ссылку на гошную репу

timoninas committed:
RT-8: Исправить описание

Убрать путаницу с разделением ролей пользователей
Исправить неопределенный статус в заголовках

timoninas committed:
RT-8: Исправить описание

Добавить заголовки с API, Бекенд с ссылкой не бек

timoninas committed:
RT-8: Убрать букву "a"
